bramkouwen/Neatly-In-Motion
# Adding Main Content to the how_to File

## What does this PR do?
* It creates the main content for how_to.html

## How do you check this PR?
* Pull down the branch
* Check the added content to how_to.html 

## Screenshots 
None.
